### PR TITLE
Trim line length in the documentation/examples for the new theme

### PR DESCRIPTION
**Pull request by JanuszL**
- the new nvidia-theme recommends the line lenght of the code snippets
  80 characters, not 100 as used in DALI
- fixes external source documentation formatting

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- the new nvidia-theme recommends the line lenght of the code snippets
  80 characters, not 100 as used in DALI
- fixes external source documentation formatting

### Affected modules and functionalities:
- documentation in docs directory
- examples
- couple of C++ files with the operators docs that include code snippets

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [x] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [x] RST
  - [x] Jupyter
  - [x] Other
- [ ] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by szalpal** [comment]
An idea: how about splitting this contribution into 3 commits, which will not be squashed before merging:
1. Adjustments in code-formatting tools (i.e. `lint.yml`, `lint.cmake`, etc...)
2. Commit which actually adjust the formatting (including auto-format and manual changes)
3. Commit that adds a line to [.git-blame-ignore-revs](https://github.com/NVIDIA/DALI/blob/main/.git-blame-ignore-revs) file, to exclude the formatting changes from blame history

**Review by szalpal — APPROVED** [review]

File: dali/operators/generic/erase/erase.cc [hunk 1]
```diff
@@ -90,8 +93,11 @@ coordinates. Since no axis_names is provided, the anchor and shape must contain
 except "C" (channels).
 What gives::
 
-    output[y, x, c] = 0               if 0.15 * 300 <= x < (0.15 * 300) + 20 and (0.15 * 300) <= y < (0.15 * 300) + 30
-    output[y, x, c] = input[y, x, c]  otherwise
+    if 0.15 * 300 <= x < (0.15 * 300) + 20 and
+       (0.15 * 300) <= y < (0.15 * 300) + 30:
+      output[y, x, c] = 0
+    else:
+      output[y, x, c] = input[y, x, c]  otherwise
```
Comment (mzient) [inline dali/operators/generic/erase/erase.cc:100]: If you make it look _that_ much like Python, then it better be Python.
```suggestion
    if (0.15 * 300 <= x < (0.15 * 300) + 20 and
       (0.15 * 300) <= y < (0.15 * 300) + 30):
      output[y, x, c] = 0
    else:
      output[y, x, c] = input[y, x, c]
```

File: dali/operators/image/color/brightness_contrast.cc [hunk 2]
```diff
@@ -79,7 +79,8 @@ DALI_SCHEMA(BrightnessContrast)
 
 The brightness and contrast are adjusted based on the following formula::
 
-  out = brightness_shift * output_range + brightness * (contrast_center + contrast * (in - contrast_center))
+  out = brightness_shift * output_range + brightness *
+        (contrast_center + contrast * (in - contrast_center))
```
Comment (mzient) [inline dali/operators/image/color/brightness_contrast.cc:83]:
```suggestion
  out = brightness_shift * output_range +
        brightness * (contrast_center + contrast * (in - contrast_center))
```

File: dali/operators/generic/erase/erase.cc (hunk 1 above)
Comment (JanuszL) [reply dali/operators/generic/erase/erase.cc:100]: Done

File: dali/operators/image/color/brightness_contrast.cc (hunk 2 above)
Comment (JanuszL) [reply dali/operators/image/color/brightness_contrast.cc:83]: Done

**Comment by JanuszL** [comment]
> An idea: how about splitting this contribution into 3 commits, which will not be squashed before merging:
> 
> 1. Adjustments in code-formatting tools (i.e. `lint.yml`, `lint.cmake`, etc...)
> 2. Commit which actually adjust the formatting (including auto-format and manual changes)
> 3. Commit that adds a line to [.git-blame-ignore-revs](https://github.com/NVIDIA/DALI/blob/main/.git-blame-ignore-revs) file, to exclude the formatting changes from blame history

Let me split this into two self contained PRs:
1) manual changes in one commit, and docs auto reformat in a second. Should be easier to review and still after squash it is self contained and passing CI
2) git-blame-ignore for the previous PR

How about that?

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
Before:
image
After:
image

File: docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb [hunk 3]
```diff
@@ -236,7 +239,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "/usr/local/lib/python3.8/dist-packages/nvidia/dali/include\n"
+      "/home/jlisiecki/Dali/dali/compile/dali/python/nvidia/dali/include\n"
```
Comment (jantonguirao) [inline docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb:242]: revert those paths(?)

File: docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb [hunk 4]
```diff
@@ -43,47 +43,47 @@
...
-      "};\n",
-      "\n",
-      "}  // namespace other_ns\n",
-      "\n",
-      "#endif  // EXAMPLE_DUMMY_H_\n"
+      "#ifndef EXAMPLE_DUMMY_H_\r\n",
+      "#define EXAMPLE_DUMMY_H_\r\n",
+      "\r\n",
+      "#include <vector>\r\n",
+      "\r\n",
```
Comment (jantonguirao) [inline docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb:50]: accidental line ending change?

File: docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb (hunk 3 above)
Comment (JanuszL) [reply docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb]: Regenerated with a different installation.

File: docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb (hunk 4 above)
Comment (JanuszL) [reply docs/examples/custom_operations/custom_operator/create_a_custom_operator.ipynb]: No idea, manually edited now.

**Review by jantonguirao — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Review by mzient — APPROVED** [review]

**Pull request merged**